### ISSUE TITLE
NVDA does not announce the error message when an invalid password is entered after activating the login button in Windows

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#20309]
### Brief summary

**Pre-Requisite:** 
1. Turn on Screen Readers: 
    NVDA: Ctrl + Alt + N
2.Verbosity:
   NVDA: Default

**Windows Test Environment:** 
OS: Windows 11 Enterprise Insider Preview Version 25H2（OS Build 26200.8246）
Browser: Microsoft Edge Version 147.0.3912.98
Screen Reader: NVDA (2026.1)
URL: 

**Observation:**
1. Screen reader is announcing as 'Loading, sign in incorrect email or password'.

**User Impact:**
Screen reader users impacted as they experience a complete lack of error feedback, as NVDA does not announce the error message when an invalid password is entered, preventing them from understanding the cause of the failure, resulting in confusion, repeated unsuccessful login attempts, increased cognitive load, and inability to complete the login process efficiently.

### Steps to reproduce

1. Open URL:  in latest edge browser.
2. Begin Registration page will be displayed.
3. Press tab key to reach 'Already Registered? View or modify your existing registration' link and activate with enter key.
4. Sign in page will be opened.
5. Press tab key to reach 'Email address edit field' and provide email address.
6. Press tab key to reach 'Password edit field' provide invalid password.
7. Press tab key to reach 'Login button' and activate with enter key.
8. Verify whether NVDA does not announce the error message when an invalid password is entered after activating the login button or not.

### Actual behavior

NVDA does not announce the error message when an invalid password is entered after activating the login button in Windows."
Screen reader is announcing as 'Loading page loading complete sign in '.

**Refer Attachment:**

1. NVDA does not announce the error message when an invalid password is entered after activating the login button in Windows.png

### Expected behavior

NVDA should announce the error message when an invalid password is entered after activating the login button in Windows."
Screen reader should announce as 'Loading page loading complete sign inincorrect email or password and focus should be on Password edit field.

### NVDA type

installed copy

### NVDA version

2026.1.1

### Windows version

Windows 11 Enterprise Insider Preview Version 25H2（OS Build 26200.8246）

### Name and version of other software in use when reproducing the issue

Microsoft Edge Version 147.0.3912.98

### Does the issue still occur after restarting your computer?

I have not restarted my computer

### If NVDA add-ons are disabled, is your problem still occurring?

I have not restarted NVDA with add-ons disabled

### Does the issue still occur after you run the System Accessibility Repair Tool (COM Registration Fixing Tool in older versions) in NVDA's tools menu?

I have not run the System Accessibility Repair Tool

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).

**Comment by msftedad** [comment]
Closing this issue.
